contents.json

```diff
@@ -15675,6 +15675,31 @@
       ],
       "date_added": "Feb 12 2018",
       "suggested_by": "@vpeschenkov"
+    },
+    {
+      "title": "Go Flashcards",
+      "category-ids": [
+        "education"
+      ],
+      "description": "Go Flashcards for iOS and WatchOS allows users to create stacks of flashcards and review them on their iPad, iPhone, or Apple Watch. All stacks and stack progress sync over CloudKit.",
```

Comment:
Could you remove `iOS` from the description, according to the point `Avoid iOS or open-source in description as it is assumed`?